packages/experience-builder-sdk/src/blocks/preview/CompositionBlock.tsx

```diff
@@ -220,7 +221,11 @@ export const CompositionBlock = ({
       ? node.children.map((childNode: ComponentTreeNode, index) => {
           return (
             <CompositionBlock
-              getPatternChildNodeClassName={_getPatternChildNodeClassName}
+              getPatternChildNodeClassName={
+                isAssembly || getPatternChildNodeClassName
```

Comment:
same deal here. This feels nonsensical and like its covering up deeper issues. Cool that it works for now, but we should refactor.